composer.json

```diff
@@ -51,6 +50,7 @@
         "oriceon/toastr-5-laravel": "dev-master",
         "pear/console_color2": "^0.1",
         "pear/console_table": "^1.3",
+        "pear/net-dns2": "*",
```

Comment:
this is not correct, you didnt run composer require, and missing composer.lock changes.

Reply:
now with exact result of:
`su librenms -c "./scripts/composer_wrapper.php require"`